src/lib/extract/extractProps.ts

```diff
@@ -111,6 +117,18 @@ export default function extractProps(
     extracted.name = String(id);
   }
 
+  if (testID) {
+    extracted.testID = String(testID);
```

Comment:
Since the types for those are already string/boolean, I think we don't need to do `String(x)` or `Boolean(x)`.